src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -156,7 +156,7 @@ func runClientProxy(cfg *config.Config) {
 	proxiedsites.Configure(cfg.ProxiedSites)
 
 	// launching geolookup service when in client mode.
-	go geolookup.StartService()
+	go geolookup.StartService(hqfd.DirectHttpClient())
```

Comment:
Great addition, we'll surely need this to proxy requests to other services in the future.